content/applications/marketing/events/sell_tickets.rst

```diff
@@ -1,68 +1,136 @@
-============
-Sell tickets
-============
+==================
+Sell event tickets
+==================
 
-Create custom ticket tiers (with various price points) for potential attendees to choose from,
-directly on the event template form, under the :guilabel:`Tickets` tab. Odoo simplifies the
-ticket-purchasing process by providing plenty of payment method options.
+Not only does Odoo *Events* provide users with the ability to create custom event tickets (and
+ticket tiers), with various price points, it also allows them to sell event tickets in two
+different ways: via standard sales orders and online through an integrated website.
+
+Odoo also simplifies the ticket-purchasing process by providing plenty of payment method options.
+
+.. tip::
+   To learn more about how to create custom tickets (and ticket tiers) for events, check out the
+   :doc:`create_events` documentation.
 
 Configuration
 =============
 
-First, in order to enable the creation (and selling of) event tickets, go to
-:menuselection:`Configuration --> Settings`, then enable the :guilabel:`Tickets` and
-:guilabel:`Online Ticketing` features.
+In order to sell event tickets in Odoo, some settings must first be enabled.
+
+First, navigate to :menuselection:`Events app --> Configuration --> Settings`, and in the
+:guilabel:`Registration` section, there are two different settings: :guilabel:`Tickets` and
+:guilabel:`Online Ticketing`.
 
-The :guilabel:`Tickets` feature allows tickets to be sold for an event.
+The :guilabel:`Tickets` setting allows users to sell event tickets with standard sales orders.
 
-The :guilabel:`Online Tickets` feature allows for the sale of tickets to occur through the website.
+The :guilabel:`Online Ticketing` setting allows users to sell event tickets online through their
+integrated Odoo website.
+
+To activate one (or both) of these settings, tick the checkbox beside the desired feature(s), and
```

Comment:
```suggestion
To activate one (or both) of these settings, tick the checkbox beside the desired features, and
```
It's my understanding we shouldn't use the (s) anymore, but if I'm wrong, ignore this change